ckan/tests/cli/test_file.py

```diff
@@ -0,0 +1,267 @@
+from __future__ import annotations
+
+from typing import Any
+
+import pytest
+from faker import Faker
+
+from ckan import logic, types
+from ckan.cli.cli import ckan
+from ckan.lib import files
+from ckan.tests.helpers import CKANCliRunner, call_action
+
+
+@pytest.fixture
+def with_temporal_storage(
```

Comment:
```suggestion
def with_temp_storage(
```
temporal means related to time :smile: